### ISSUE TITLE
Dropdown: Error message referenced by aria-describedby was not announced

### ISSUE THREAD

**Issue by ssaniel** [infor-design/enterprise#5130]
**Describe the bug**
Error message referenced by aria-describedby was not announced. 

**To Reproduce**
1) Open up any form
2) Supply invalid input to a dropdown
3) Save (Error message should be visible)
4) press 'Tab' to focus the dropdown

**Expected behavior**
Error message of the dropdown should be announced.

**Version**
ids-enterprise: 4.50.2

**Platform**
Infor Application/Team Name: HCM
OS Version: Windows 10
Browser Name: Chrome
Browser Version: Version 90.0.4430.85 (Official Build) (64-bit)

**Additional context**
Probably the reason why screen reader cannot announce the error message is because the focused element in the dropdown is the <div> not <select> and aria-describedby was added in <select>. Please see attached screenshot
<img width="856" alt="dropdown" src="https://user-images.githubusercontent.com/78124324/115855974-71ea6580-a45e-11eb-8a4b-104eb993f6f3.png">

**Comment by tmcconechy** [comment]
@ssaniel It seems ok on https://master-enterprise.demo.design.infor.com/components/dropdown/example-index.html

The code that is putting in the label for some reason must be going wrong on your end. Maybe you can debug whats happening here https://github.com/infor-design/enterprise/blob/main/src/components/dropdown/dropdown.js#L493

**Comment by tmcconechy** [comment]
This can be reproduced on https://master-enterprise.demo.design.infor.com/components/dropdown/example-validation.html

- select the blank item
- tab back in the field

Need to somehow connect the label ? maybe add aria-described by or append to the aria-label.

**Comment by jbrcna** [comment]
This has been QA tested and passed on v4.56.0-dev.
verified in https://main-enterprise.demo.design.infor.com/components/dropdown/example-validation.html
![image](https://user-images.githubusercontent.com/56722184/134482379-c8136286-962c-4ea0-8fa6-f3f20df31e61.png)
